components/mediaUploader.css

```diff
@@ -123,3 +123,6 @@
   text-align: center;
   opacity: 1;
 }
+.md-block.md-icon-float.md-input-has-placeholder > label{
+  display:none!important;
```

Comment:
We don't use `!important` in css.  This also has merge conflicts with the master branch.